.gitignore

```diff
@@ -6,3 +6,4 @@ Cargo.lock
 .vscode
 components_esp32*.lock
 .envrc
+/build
```

Comment:
why this addition in the ignores?

Reply:
CMake kept generating a build directory because of the C code and I was tired of accidentally committing it